app/editor/src/features/content/papers/Papers.tsx

```diff
@@ -90,17 +89,21 @@ const Papers: React.FC<IPapersProps> = (props) => {
   }, [onContentUpdated, hub]);
 
   const handleClickUse = React.useCallback(
-    (content: IContentModel) => {
-      updateStatus(content)
+    (content: IContentSearchResult) => {
+      getContent(content.id)
```

Comment:
I don't like this solution.  But it temporarily resolves the issue for updating the status of the selected content.